[Bug] cmake fails to find or compile RncLib for Animview

#### Describe the issue

Cmake doesn't compile RncLib for Animview

Version
``` bash
$ cmake --version
cmake version 3.16.3
```

Build output
``` bash
$ mkdir build
$ cd build
$ cmake ..
-- The C compiler identification is GNU 9.3.0
-- The CXX compiler identification is GNU 9.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found wxWidgets: -L/usr/lib/x86_64-linux-gnu;-pthread;;;-lwx_gtk3u_xrc-3.0;-lwx_gtk3u_html-3.0;-lwx_gtk3u_qa-3.0;-lwx_gtk3u_adv-3.0;-lwx_gtk3u_core-3.0;-lwx_baseu_xml-3.0;-lwx_baseu_net-3.0;-lwx_baseu-3.0 (found version "3.0.4") 
  wxWidgets found
CMake Warning (dev) in CMakeLists.txt:
  No cmake_minimum_required command is present.  A line of code such as

    cmake_minimum_required(VERSION 3.16)

  should be added at the top of the file.  The version specified may be lower
  if you wish to support older CMake versions for this project.  For more
  information run "cmake --help-policy CMP0000".
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Configuring done
-- Generating done
-- Build files have been written to: ..../AnimView/build

$ make
Scanning dependencies of target AnimView
[ 40%] Building CXX object CMakeFiles/AnimView.dir/frmSprites.o
[ 40%] Building CXX object CMakeFiles/AnimView.dir/th.o
[ 80%] Building CXX object CMakeFiles/AnimView.dir/frmMain.o
[ 80%] Building CXX object CMakeFiles/AnimView.dir/app.o
[100%] Linking CXX executable AnimView
/usr/bin/ld: cannot find -lRncLib
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/AnimView.dir/build.make:129: AnimView] Error 1
make[1]: *** [CMakeFiles/Makefile2:76: CMakeFiles/AnimView.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```

--- a/AnimView/CMakeLists.txt
+++ b/AnimView/CMakeLists.txt
@@ -1,3 +1,9 @@
+# Sanity check
+if(CORSIX_TH_DONE_TOP_LEVEL_CMAKE)
+else()
+  message(FATAL_ERROR "Please run CMake from the top-level directory instead of here.")
+endif()
+
 # Project Declaration
 project(AnimView)
 

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -14,7 +14,7 @@
 #   - WITH_LUAROCKS : Install required luarocks in the macOS app (requires luarocks)
 
 #   - ENABLE_UNIT_TESTS : Enable Unit Testing Target (requires Catch2) (no)
-#   - BUILD_ANIMVIEWER : Generate AnimViewer build files (no)
+#   - BUILD_ANIMVIEW : Generate AnimViewer build files (no)
 
 cmake_minimum_required(VERSION 3.5)
 
@@ -59,7 +59,7 @@ if(APPLE)
 endif()
 
 option(ENABLE_UNIT_TESTS "Enables Unit Testing Targets" OFF)
-option(BUILD_ANIMVIEWER "Build the animation viewer as part of the build process" OFF)
+option(BUILD_ANIMVIEW "Build the animation viewer as part of the build process" OFF)
 
 if(WITH_AUDIO)
   set(CORSIX_TH_USE_SDL_MIXER ON)
@@ -129,7 +129,7 @@ add_subdirectory("libs")
 message("Building CorsixTH")
 add_subdirectory(CorsixTH)
 
-if(BUILD_ANIMVIEWER)
+if(BUILD_ANIMVIEW)
   message("Building AnimView")
   add_subdirectory(AnimView)
 endif()

--- a/CorsixTH/CMakeLists.txt
+++ b/CorsixTH/CMakeLists.txt
@@ -1,7 +1,7 @@
 # Sanity check
 if(CORSIX_TH_DONE_TOP_LEVEL_CMAKE)
 else()
-  message(FATAL_ERROR "Please run cmake on the top-level directory, not this one.")
+  message(FATAL_ERROR "Please run CMake from the top-level directory instead of here.")
 endif()
 
 # Project Declaration

